docs/_pages/how-to-guide.md

```diff
@@ -7,15 +7,28 @@ toc: true
 # Prerequisites
 * [terraform >= 0.12](https://www.terraform.io/)
   - credentials configured for an AWS account
+  - note, some modules are pinned to 0.14 It is advised to use this version for now for best compatability. 
 * [terragrunt >= v0.21](https://github.com/gruntwork-io/terragrunt)
+* [AWS CLI >= 2.0](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
```

Comment:
Mark this optional and remove the version. It is not a general requirement for AWS auth (though it can be required for a specific implementation environment if using AWS SSO). And any version is fine (again, same exception). We can cover some base case using the awscli for AWS in particular, but should still be clear it is an example and point at the provider docs for details on how to set up authentication to the cloud of choice.

Reply:
updated clarification to point to provider.